subprojects/common/elastic/src/main/kotlin/com/avito/android/elastic/ElasticLogEventRequest.kt

```diff
@@ -0,0 +1,13 @@
+package com.avito.android.elastic
+
+import com.google.gson.annotations.SerializedName
+
+@Suppress("unused")
+internal class ElasticLogEventRequest(
+    @SerializedName("@timestamp") val timestamp: String,
+    val tag: String,
+    val level: String,
+    val buildId: String,
+    val message: String,
+    val errorMessage: String?
+)
```

Comment:
How about `SerializedName` and tests for API contract?
I don't want to break logging after refactoring this model.